sn_protocol/src/error.rs

```diff
@@ -88,25 +60,19 @@ pub enum Error {
     /// Received a `Vec<SignedSpend>` with more than two spends
     #[error("Incoming SpendDbc PUT with incorrect number of SignedSpend")]
     MaxNumberOfSpendsExceeded,
-    /// Spend not found.
```

Comment:
Why have we removed _some_ comments here?

Reply:
The comments where identical to the error name, should we put them back?